textarea/FCKeditor/connector/src/java/org/sakaiproject/connector/fck/FCKConnectorServlet.java

```diff
@@ -373,6 +373,8 @@ else if ("CreateFolder".equals(commandStr)) {
      }
      
 
+    private static final int MAX_SAVE_RETRIES = 1000;
```

Comment:
I would argue for 100 based on this existing piece of code:

api/src/main/java/org/sakaiproject/content/api/ContentHostingService.java:	public static final int MAXIMUM_ATTEMPTS_FOR_UNIQUENESS = 100;